remote_manager/include/remote_manager/remote_manager.h

```diff
@@ -0,0 +1,51 @@
+#ifndef REMOTE_MANAGER_H
+#define REMOTE_MANAGER_H
+
+#include <ros/ros.h>
+#include <time.h>
+
+#include <geometry_msgs/Twist.h>
+
+#include <std_msgs/Int32.h>
+#include <std_msgs/Bool.h>
+
+class RemoteManager
+{
+    public:
+        RemoteManager(ros::NodeHandle &nh, ros::NodeHandle &pn);
+        void init();
+        void update();
+
+        void Drive(const geometry_msgs::Twist& cmd_vel_used);
+        void Stop();
+        void Pause();
```

Comment:
For consistent naming of functions.

```suggestion
        void drive(const geometry_msgs::Twist& cmd_vel_used);
        void stop();
        void pause();
```